src/endpoints/sprungbrettJobs.js

```diff
@@ -8,14 +8,12 @@ import type { JsonSprungbrettJobType } from '../types'
 
 const SPRUNGBRETT_JOBS_ENDPOINT_NAME = 'sprungbrettJobs'
 
-type ParamsType = { url: ?string }
-
-const endpoint: Endpoint<ParamsType, Array<SprungbrettJobModel>> = new EndpointBuilder(SPRUNGBRETT_JOBS_ENDPOINT_NAME)
-  .withParamsToUrlMapper((params): string => {
-    if (!params.url) {
-      throw new ParamMissingError(SPRUNGBRETT_JOBS_ENDPOINT_NAME, 'url')
+const endpoint: Endpoint<*, Array<SprungbrettJobModel>> = new EndpointBuilder(SPRUNGBRETT_JOBS_ENDPOINT_NAME)
+  .withParamsToUrlMapper((apiUrl: string): string => {
+    if (!apiUrl) {
+      throw new ParamMissingError(SPRUNGBRETT_JOBS_ENDPOINT_NAME, 'apiUrl')
     }
-    return params.url
+    return apiUrl
```

Comment:
I don't really like, that we use the `apiUrl` here as the actual url and with the cms-endpoints it's just used as a baseUrl. Also having `apiUrl` as a parameter of the `fetch`-function in `Endpoint` makes it harder to use for apps (https://github.com/Integreat/integreat-webapp/blob/develop/src/modules/app/fetchData.js would need to be adjusted quite heavily as far as I can see).

I think we should either have baseUrl as a param for the integreat-endpoints, or we could export functions returning endpoints instead of endpoints themselves. The baseUrl could then be a param of that generator-function for the cms-endpoints.